### PR TITLE
Packages readme files

### PR DESCRIPTION
**Pull request by flashdesignory**
Readmes for all projects in the packages directories. 

Will add more details as we integrate with Speedometer.

@kara

### CODE REVIEW

File: packages/tools/app-build-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# app-build-scripts
+
+A collection of helper functions to create a static output for a workload without any bundlers.
+
+## example
+
+The following example is taken from the [todomvc-es5](../../../apps/todomvc//todomvc-es5/README.md) workload.
```
Comment (camillobruni) [inline packages/tools/app-build-scripts/README.md:7]: is the double `//` on purpose?

Comment (flashdesignory) [reply packages/tools/app-build-scripts/README.md:7]: 🤦

File: README.md [hunk 2]
```diff
@@ -292,7 +292,6 @@ Several interactions are available, such as:
 pnpm -F news-site-next dev
 pnpm -F news-site-next build
 pnpm -F news-site-next build:static
-pnpm -F news-site-next build:gh
```
Comment (flashdesignory) [inline README.md:295]: removed these scripts since they are too specific to github pages.

**Review by camillobruni — APPROVED** [review]

File: packages/styles/todomvc-css/README.md [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
 
-install the package
+## How to install
+
+Install the package manually, by adding an entry to your package.json file and running `pnpm install`.
 
 ```bash
-npm install todomvc-app-css-alt
+"dependencies": {
+    "todmvc-css": "workspace:*"
```
Comment (kara) [inline packages/styles/todomvc-css/README.md:18]:
```suggestion
    "todomvc-css": "workspace:*"
```

File: packages/tools/benchmark-connector/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# benchmark-connector
+
+The benchmark-connector enables communication between the benchmark and a workload via the `post-message` api.
+Each workload that wants to opt in, needs the `benchmark-connector.min.js` file added to its html pages.
```
Comment (kara) [inline packages/tools/benchmark-connector/README.md:4]:
```suggestion
Each workload that wants to opt in needs the `benchmark-connector.min.js` file added to its html pages.
```

File: packages/tools/benchmark-connector/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+A `prepare` file is available, that can copy the benchmark-connector.min.js file from the node-modules folder to the appropriate directory of the workload. This is typically the `public`, `dist` or the `root` folder and is customizable through the `HOST` environment variable of the npm script. During the prepare process, two global variables get set and added to the benchmark-connector.min file:
+
+-   window.name: Package name of the workload.
+-   window.version: Package version of the workload.
+
+Both variables are used to create an `appId`, which will get send with each postMessage event.
```
Comment (kara) [inline packages/tools/benchmark-connector/README.md:11]:
```suggestion
Both variables are used to create an `appId`, which will get sent with each postMessage event.
```

**Review by kara — APPROVED** [review]
LGTM aside from typos

**Pull request merged**